### PR TITLE
Fix selection of not parsable file should not change query path

### LINKED ISSUE #335: Switch to draw view after opening a file that is not part of a diagram in text view
**Describe the bug**

In the text view, when I open a file that is not part of a diagram (example: `README.md`), then switch to the draw view: the draw view is broken and does not show the drawers.

image

* There are no errors in the console.
* The URL of the draw view is: http://localhost:8080/#/projects/myproj/modelizer/draw.
    * It is missing the query string parameters: `?plugin=terrator-plugin&path=mydiag`.

This is because of the feature which allows switching to another diagram by opening a file in the text view.

**To Reproduce**

https://github.com/ditrit/leto-modelizer/assets/20184007/c0f42c2b-b81c-4bd7-9d7f-4f77e0bc4410

**Expected behavior**

Show the previously opened diagram in the draw view (same behaviour as when all tabs are closed in the text view).

**Application**

- leto-modelizer commit hash: `1c56e93`.

### PR DESCRIPTION
**Pull request by Zorin95670**
Bug appear due to new diagram management. This pr should fix this bug.

Resolved #335

### CODE REVIEW

**Review by justinehell — DISMISSED** [review]

File: tests/unit/layouts/ModelizerTextLayout.spec.js [hunk 1]
```diff
@@ -110,6 +112,61 @@ describe('Test component: ModelizerTextLayout', () => {
...
+        getModels: () => ([]),
+        data: {
+          name: 'test',
+        },
+      }]));
+      getPluginByName.mockImplementation(() => ({
+        isParsable: () => false,
+        getModels: () => ([]),
+        data: {
+          name: 'test2',
```
Comment (k-y0u) [inline tests/unit/layouts/ModelizerTextLayout.spec.js:152]: replace `test2` by `wrapper.vm.query.plugin` to prove the test

File: tests/unit/layouts/ModelizerTextLayout.spec.js [hunk 2]
```diff
@@ -110,6 +112,61 @@ describe('Test component: ModelizerTextLayout', () => {
...
+
+      expect(useRouterPush).toHaveBeenCalledTimes(1);
+      expect(useRouterPush).toHaveBeenCalledWith({
+        name: 'Text',
+        params: {
+          projectName: wrapper.vm.projectName,
+        },
+        query: {
+          path: 'pluginName/modelName',
+          plugin: 'test2',
```
Comment (k-y0u) [inline tests/unit/layouts/ModelizerTextLayout.spec.js:166]: replace `test2` by `wrapper.vm.query.plugin` to prove the test

File: tests/unit/layouts/ModelizerTextLayout.spec.js [hunk 3]
```diff
@@ -110,6 +112,61 @@ describe('Test component: ModelizerTextLayout', () => {
...
+      await wrapper.vm.onSelectFileTab('plugin/fileName');
+
+      expect(useRouterPush).toHaveBeenCalledTimes(1);
+      expect(useRouterPush).toHaveBeenCalledWith({
+        name: 'Text',
+        params: {
+          projectName: wrapper.vm.projectName,
+        },
+        query: {
+          path: 'pluginName/modelName',
```
Comment (k-y0u) [inline tests/unit/layouts/ModelizerTextLayout.spec.js:134]: replace `test` by `wrapper.vm.query.path` to prove the test

File: cypress/e2e/ModelizerPage/TextView/file/SwitchModelFile.feature [hunk 4]
```diff
@@ -55,3 +55,43 @@ Feature: Test modelizer text view: switch model file
...
+    When I click on '[data-cy="modelizer-switch-button"] [aria-pressed="false"]'
+    And I wait 2 seconds
+    Then I expect current url is '{{ projectName }}/modelizer/text\?plugin=terrator-plugin&path={{ modelFolder }}'
+    And  I expect '[data-cy="file-explorer"] [data-cy="folder_{{ projectName }}"]' is '{{ projectName }}'
+    And  I expect '[data-cy="file-explorer"] [data-cy="folder_{{ modelFolder }}"]' exists
+    And  I expect '[data-cy="file-explorer"] [data-cy="file_README.md"]' exists
+    And  I expect '[data-cy="file-tabs-container"] [data-cy="active-tab"]' is 'main.tf'
+
+    When I double click on '[data-cy="file-explorer"] [data-cy="file_README.md"]'
+    Then I expect current url is '{{ projectName }}/modelizer/text\?plugin=terrator-plugin&path={{ modelFolder }}'
```
Comment (k-y0u) [inline cypress/e2e/ModelizerPage/TextView/file/SwitchModelFile.feature:96]: reverse `expect current url` line and `I expect '[data-cy="file-tabs-container"]` line

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: tests/unit/layouts/ModelizerTextLayout.spec.js (hunk 1 above)
Comment (Zorin95670) [reply tests/unit/layouts/ModelizerTextLayout.spec.js:152]: done

File: tests/unit/layouts/ModelizerTextLayout.spec.js (hunk 2 above)
Comment (Zorin95670) [reply tests/unit/layouts/ModelizerTextLayout.spec.js:166]: done

File: tests/unit/layouts/ModelizerTextLayout.spec.js (hunk 3 above)
Comment (Zorin95670) [reply tests/unit/layouts/ModelizerTextLayout.spec.js:134]: done

File: cypress/e2e/ModelizerPage/TextView/file/SwitchModelFile.feature (hunk 4 above)
Comment (Zorin95670) [reply cypress/e2e/ModelizerPage/TextView/file/SwitchModelFile.feature]: done

**Review by justinehell — APPROVED** [review]

**Pull request merged**